lib/spack/spack/test/packages.py

```diff
@@ -114,6 +125,8 @@ def test_import_package_as(self):
         import spack.pkg.builtin.mock as m              # noqa
         from spack.pkg.builtin import mock              # noqa
 
+    @pytest.mark.skipif(sys.platform == 'win32',
+                        reason="Not supported on Windows (yet)")
     def test_inheritance_of_diretives(self):
```

Comment:
To follow up: I wanted to confirm that in general we can keep skipping these tests. It would generally be good to list out the features that aren't working in the PR description though; e.g. in this case "Package inheritance with repositories does not work (build system inheritance does work)"

Reply:
In general, we are ok to be skipping these tests, and other like them. They indicate that, at one point this feature was failing on Windows, and has yet to be fixed, or the tests yet to be revisited (which is an ongoing effort at this time), however, as all tests run on Unix, we can confirm that the Windows contributions will not break any current Spack features on Unix.

I am currently compiling a document, that we will include in the getting started on Windows documentation, cataloguing, as best as is possible, the currently unsupported features on the Windows branch.

Reply:
To answer your earlier question, this test may have been failing at one point, and has not yet been revisited. I'll try to verify what the status of this test is now.

Reply:
Apologies: I marked this resolved but was going through this again and noticed that the skip statement is still there and I don't see an explanation (I've mentioned elsewhere that detailed explanations are generally not required for individual test failures but in this case I'd like to know more).